listenbrainz/db/lovehate.py

```diff
@@ -0,0 +1,138 @@
+import sqlalchemy
+
+from listenbrainz import db
+
+
+def insert_lovehate_record(user_id, recording_msid, score):
```

Comment:
I suppose at some point we should also change this to insert_feedback_record. And if the module is called feedback, then this could just be called insert(). Pretty simple...

Reply:
Makes sense. I m already on it